lib/runtime-core/src/sys/windows/memory.rs

```diff
@@ -45,7 +46,10 @@ impl Memory {
         let ptr = unsafe { VirtualAlloc(ptr::null_mut(), size, flags, protect) };
 
         if ptr.is_null() {
-            Err("unable to allocate memory".to_string())
+            Err(format!(
+                "unable to allocate memory (error code: %d)",
```

Comment:
What's up with the `%d` error formatting here?

does this compile?

Reply:
Oopsy :-p